### PR TITLE
#165716752 Setup Authors Heaven for Heroku deployment

### PR DESCRIPTION
**Pull request by patrickf949**
#### What does this PR do?

Sets up Authors Heaven for Deployment on Heroku

#### Description of Tasks to be done 

- Add and configure procfile to the root of the project
- Configure server.js file
- Install express 
- Configure node in the package.json

#### How can this be tested?

Checkout the heroku link [here](https://ah-frontend-prime.herokuapp.com/)

#### Screenshot

Screenshot 2019-05-02 at 13 56 34

#### Relevant Pivotal tracker stories
[#165716752](https://www.pivotaltracker.com/story/show/165716752)

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: node server.js
```
Comment (e-ian) [inline Procfile]: @patrickf949 add a new line at the end of this file and elsewhere

**Review by e-ian — APPROVED** [review]
Tested. LGTM

**Review by anyatibrian — APPROVED** [review]
tested and it works fine great work

**Review by Genza999 — APPROVED** [review]
Testes and LGTM. Great work @patrickf949.

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Pull request merged**